OSDOCS-10446 OCP 4.14.24 RNs (#75560)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
====

[id="ocp-4-14-24"]
=== RHSA-2024:2668 - {product-title} 4.14.24 bug fix update and security update

Issued: 2024-05-09

{product-title} release 4.14.24, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:2668[RHSA-2024:2668] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:2672[RHSA-2024:2672] advisory.

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.14.24 --pullspecs
----

[id="ocp-4-14-24-enhancements"]
==== Enhancements

The following enhancements are included in this z-stream release:

[id="ocp-4-14-ipv6-default-macvlan"]

> ==== IPv6 unsolicited neighbor advertisements now default on macvlan CNI plugin

Reviewer: ```suggestion
==== IPv6 unsolicited neighbor advertisements now default on macvlan CNI plug-in
```

> ==== IPv6 unsolicited neighbor advertisements now default on macvlan CNI plug-in

Reviewer 2: currently same level as its parent
```suggestion
===== IPv6 unsolicited neighbor advertisements now default on macvlan CNI plug-in
```

> Pods created using the macvlan CNI plugin, where the IP address management CNI plug-in has assigned IPs, now send IPv6 unsolicited neighbor advertisements by default onto the network. This notifies hosts of the new pod's MAC address for a particular IP to refresh IPv6 neighbor caches. (link:https://issues.redhat.com/browse/OCPBUGS-33066[*OCPBUGS-33066*])

Reviewer: ```suggestion
Pods created using the macvlan CNI plug-in, where the IP address management CNI plug-in has assigned IP addresses, now send IPv6 unsolicited neighbor advertisements by default onto the network. This notifies hosts of the new pod's MAC address for a particular IP address to refresh IPv6 neighbor caches. (link:https://issues.redhat.com/browse/OCPBUGS-33066[*OCPBUGS-33066*])
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#p

> Pods created using the macvlan CNI plug-in, where the IP address management CNI plug-in has assigned IP addresses, now send IPv6 unsolicited neighbor advertisements by default onto the network. This notifies hosts of the new pod's MAC address for a particular IP address to refresh IPv6 neighbor caches. (link:https://issues.redhat.com/browse/OCPBUGS-33066[*OCPBUGS-33066*])

Reviewer 2: I noticed in [another PR](https://github.com/openshift/openshift-docs/pull/75528) that these items had bullets like so
```suggestion
* Pods created using the macvlan CNI plug-in, where the IP address management CNI plug-in has assigned IP addresses, now send IPv6 unsolicited neighbor advertisements by default onto the network. This notifies hosts of the new pod's MAC address for a particular IP address to refresh IPv6 neighbor caches. (link:https://issues.redhat.com/browse/OCPBUGS-33066[*OCPBUGS-33066*])
```

[id="ocp-4-14-24-bug-fixes"]
==== Bug fixes

> * Previously, when a cluster was installed using proxy and the proxy information contained escape characters in the format "%XX" the installation would fail. With this release, the issue has been fixed. (link:https://issues.redhat.com/browse/OCPBUGS-33010[*OCPBUGS-33010*])

Reviewer 2: ```suggestion
* Previously, when a cluster was installed using proxy and the proxy information contained escape characters in the format `%XX` the installation would fail. With this release, the issue has been fixed. (link:https://issues.redhat.com/browse/OCPBUGS-33010[*OCPBUGS-33010*])
```

* Previously, in hosted control planes for {product-title}, when you created the custom resource definition (CRD) for `ImageDigestMirrorSet` and `ImageContentSourcePolicy` objects at the same time in a disconnected environment, the HyperShift Operator created the object only for the `ImageDigestMirrorSet` CRD, ignoring the `ImageContentSourcePolicy` CRD. With this release, the HyperShift Operator creates objects for the `ImageDigestMirrorSet` and `ImageContentSourcePolicy` CRDs. (link:https://issues.redhat.com/browse/OCPBUGS-32471[*OCPBUGS-32471*])

> * Previously, nodes of paused MachineConfigPools may be incorrectly unpaused when performing a cluster update. With this update, Nodes of paused MachineConfigPools correctly stay paused when performing a cluster update. (link:https://issues.redhat.com/browse/OCPBUGS-32168[*OCPBUGS-32168*])

Reviewer: ```suggestion
* Previously, nodes of paused MachineConfigPools might be incorrectly unpaused when performing a cluster update. With this update, nodes of paused MachineConfigPools correctly stay paused when performing a cluster update. (link:https://issues.redhat.com/browse/OCPBUGS-32168[*OCPBUGS-32168*])
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m

* Previously, the image registry did not support {aws-first} region `ca-west-1`. With this release, the image registry can now be deployed in this region. (link:https://issues.redhat.com/browse/OCPBUGS-31857[*OCPBUGS-31857*])

> * Previously, Terraform would create the compute server group with the policy set for the control plane. As a consequence, the 'serverGroupPolicy' property of the install-config.yaml file was ignored for the compute server group. With this release, the server group policy set in the install-config.yaml file for the compute MachinePool is correctly applied at install-time in the Terraform flow. (link:https://issues.redhat.com/browse/OCPBUGS-31756[*OCPBUGS-31756*])

Reviewer 2: ```suggestion
* Previously, Terraform would create the compute server group with the policy set for the control plane. As a consequence, the `serverGroupPolicy` property of the `install-config.yaml` file was ignored for the compute server group. With this release, the server group policy set in the `install-config.yaml` file for the compute machine pool is correctly applied at install-time in the Terraform flow. (link:https://issues.redhat.com/browse/OCPBUGS-31756[*OCPBUGS-31756*])
```
